README.md

```diff
@@ -91,6 +91,32 @@ dynamic_form:
 ### Options
 Field options are the same as symfony, refer to the documentation
 
+#### Birthday Type
+
+Supports `Symfony >= 4.4`.
+
+In addition to the own options that the field brings by default ([see documentation](https://symfony.com/doc/current/reference/forms/types/birthday.html)), we added 3 new attributes to make its behavior more flexible.
+
+##### MinYear and MaxYear:
+
+These options work together and it is an improvement to the `years` option because it is too strict to only receive an array with the allowed years (and that in our case we should leave static in our Yaml file). With them we can select (taking the current date) the minimum and maximum number of years that will be displayed in the form. These options only accept integers greater than or equal to zero.
+
+If we want, for example, to have a field to enter the date of birth but only those over 18 and under 120 are allowed to register, it would look like this:
+
+```yaml
+dynamic_form:
+    new_user:
+        birthday:
+            enabled: true
+            type: birthday
+            options:
+                minYear: 18
```

Comment:
I think the name is misleading, is not clear if it refers to `minYear` i.e. 1997, or like you use it as minimum years backward.  Something like `minNumberOfYearFromNow`  or something like that could work better.

Reply:
Yes, sure! I think it is clearer with `minAgeAllowed` and` maxAgeAllowed`, since the idea is to show the range of years and that they are within those values, what do you think?

I'll also add a more literal example to the documentation. For example, if we want to allow only children with between `minAgeAllowed: 5` and `maxAgeAllowed: 10` years to be registered, then an array with these years would be generated `[2016, 2015, 2014, 2013, 2012, 2011]` taking into account as the current year the `2021`.

Reply:
Sounds good!